### DISCUSSION TITLE
请问这个如何部署到自己的云服务器呢？

### DISCUSSION THREAD

**Discussion by ufoozhenghao** [ddiu8081/chatgpt-demo#293] [Q&A]
npm run build之后我开没有index.html入口文件，nginx好像用不了？应该如何部署呢？官方的教程给的没有我的云服务器商

**Comment by yzh990918** [comment]
用 node 起 server. https://github.com/ddiu8081/chatgpt-demo/discussions/254

**Reply by YoKONCy** [reply]
我用npm run build 打包之后，得到的dist文件夹里面有server和client两个文件夹，然后我用pm2启动了server中的entry.mjs，但是好像还是有问题，我通过ip加端口还是访问不了

**Reply by ddiu8081** [reply]
> 我用npm run build 打包之后，得到的dist文件夹里面有server和client两个文件夹，然后我用pm2启动了server中的entry.mjs，但是好像还是有问题，我通过ip加端口还是访问不了

@YoKONCy https://docs.astro.build/en/guides/integrations-guide/node/#standalone (Custom host and port)
确认一下是否按照文档传过 `HOST` 参数。默认运行时不支持从 ip 访问，需要指定 `HOST=0.0.0.0`

**Reply by YoKONCy** [reply]
好的，解决了，感谢

**Reply by YoKONCy** [reply]
又有一个新的问题，我这样操作之后，在我云服务器上运行之后，填了自己的秘钥，然后好像上下文连续的能力消失了，但是我在github上托管的却又没有消失

**Reply by ddiu8081** [reply]
> 又有一个新的问题，我这样操作之后，在我云服务器上运行之后，填了自己的秘钥，然后好像上下文连续的能力消失了，但是我在github上托管的却又没有消失

#166

**Reply by YoKONCy** [reply]
解决了，感谢大佬！
